src/Components/Articles.jsx

```diff
@@ -1,8 +1,30 @@
+import { useState } from "react"
+import axios from "axios"
+import ArticleCard from "./ArticleCard"
+
 function Articles() {
+    const [articles, setArticles] = useState([])
+    function handleApi() {
+        axios.get("https://nc-news-jstr.onrender.com/api/articles")
+        .then((response) => {
+            console.log(response)
+            setArticles(response.data.articles)
+        })
+        .catch((err) => {
+            console.log(err)
+        })
+    }
+    handleApi()
+
     return (
         <main>
-        <h2>Hello!</h2>
-        
+        <h2>Articles</h2>
+        <ul id="article-container">
+            {articles.map((article, index) => {
+                return <li className="article-card" key={index}> <ArticleCard article={article}/>
```

Comment:
is there a better key than the index? Perhaps a unique identifier that every article has?